ci/tests/022-cargo-test.sh

```diff
@@ -3,4 +3,4 @@
 
 set -xeo pipefail
 
-cargo $CARGOARGS test --workspace --all-features --exclude "r1cs-std"
+cargo $CARGOARGS test --workspace --all-features --exclude "r1cs-std" --exclude "proof-systems"
```

Comment:
Can't you pass a list here ?

Reply:
It seems not to be possible, the docs just mention that [`--exclude` can be employed multiple times](https://doc.rust-lang.org/cargo/commands/cargo-test.html#package-selection); the only other way mentioned in the docs would be relying on glob patterns, but they do not support `|` regular expression operator